rele/config.py

```diff
@@ -31,5 +33,9 @@ def load_subscriptions_from_paths(sub_module_paths, sub_prefix=None):
             if isinstance(attribute, Subscription):
                 if sub_prefix and not attribute.prefix:
                     attribute.set_prefix(sub_prefix)
+
+                if filter_by and not attribute.filter_by:
```

Comment:
Setting prefix and filter must be handled by `Subscription`.
Therefore I suggest moving the conditions inside `set_prefix` and `set_filter_by`.

Reply:
This is to set a default filter from the settings, I'm not sure it should be handled by the Subscription.

Reply:
This is not ideal but the only way to avoid it would be to implement global settings so that the `sub` decorator can read from them when instantiating the `Subscription` instances. I'd say let's handle that issue separately, since this PR uses the existing mechanism correctly.